compiler/src/iree/compiler/Tools/init_mlir_passes.h

```diff
@@ -57,6 +58,9 @@ inline void registerMlirPasses() {
   affine::registerAffinePipelineDataTransferPass();
   registerConvertAffineToStandardPass();
 
+  // Arm SME
+  arm_sme::registerArmSMEPasses();
```

Comment:
This kind a sucks. Why did we need this. It would be better if they were just registered within Codegen instead of it being registered all the way here.

Reply:
Codegen dialect and pass registration is a bit wonky. Could use another look at some point.